tests/data/nomenclature_configs/multiple_repos_for_mapping.yaml

```diff
@@ -5,5 +5,5 @@ repositories:
     url: https://github.com/IAMconsortium/legacy-definitions.git/
 mappings:
   repositories:
-    - common-definitions
-    - legacy-definitions
+    - name: common-definitions
```

Comment:
Is the name now required? Could cause a lot of headache of that is now mandatory…

Reply:
Yes, better to add a field-validator that translates a string to `name: {value}` to avoid breaking current workflows